public/styles/components/_select.scss

```diff
@@ -54,3 +56,30 @@
     content: '';
   }
 }
+
+.bsds-field-label {
+  &.bsds-filter-select {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+@media (min-width: $breakpoint-md-min) {
+  .bsds-field-label {
+    &.bsds-filter-select {
+      gap: space(2x);
+      flex-direction: row;
+      align-items: center;
+
+      .bsds-field-label-text {
+        max-width: calc(space(12x) + space(8x));
+        padding-top: calc(space(md) - space(xxs));
+        word-break: break-word;
```

Comment:
i lean away from calculating one-off values using the spacing scale because then we lose track of how many overrides we have (and we do something in our code that we tell other devs not to do). what are we solving for with the max-width? is there any other way to approach it using flex/grid?